include/picongpu/param/memory.param

```diff
@@ -82,6 +84,21 @@ namespace picongpu
         static constexpr uint32_t BYTES_EXCHANGE_Z = 1 * 1024 * 1024; // 1 MiB
         static constexpr uint32_t BYTES_EDGES = 32 * 1024; // 32 kiB
         static constexpr uint32_t BYTES_CORNER = 8 * 1024; // 8 kiB
+
+        /** Reference local domain size
+         *
+         * The size of the local domain for which the exchange sizes `BYTES_*` are configured for.
+         * The required size of each exchange will be calculated at runtime based on the local domain size and the
+         * reference size. The exchange size will be scaled only up and not down. Zero means that there is no reference
+         * domain size, exchanges will not be scaled.
+         */
+        using REF_LOCAL_DOM_SIZE = mCT::Int<0, 0, 0>;
+        /** Scaling rate per direction.
+         *
+         * 1.0 means it scales linear with the ratio between the local domain size at runtime and the reference local
+         * domain size.
+         */
+        const std::array<float_X, 3> DIR_SCALING_FACTOR = {0.0, 0.0, 0.0};
```

Comment:
I am not sure if these comments are very clear. Especially I find the description of `DIR_SCALING_FACTOR` is confusing if not misleading: isn't any positive value linear scaling as `f(x) = DIR_SCALING_FACTOR[...] * x`? Maybe easier to discuss via mattermost or VC